### PR TITLE
Basic Kubernetes Docs

### PR DESCRIPTION
**Pull request by jrcrawfo**
What?
Basic configuration docs for Kubernetes

Why?
Operators need to know what configs are required to run Heron on k8s

### CODE REVIEW

File: website/content/docs/operators/deployment/schedulers/kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+
+# The count of retry to verify whether seting up a tunnel process
+heron.statemgr.tunnel.verify.count:             10
+
+# SSH tunnel host
+heron.statemgr.tunnel.host:              "<user>@<host>"
+```
+
+## Working with Topologies
+
```
Comment (srkukarni) [inline website/content/docs/operators/deployment/schedulers/kubernetes.md]: Could you please add some doc about sample heron submit/kill commands on kubernetics

**Comment by kramasamy** [comment]
@jrcrawfo - is this ready to go?

**Comment by lucperkins** [comment]
@kramasamy @jrcrawfo This page has already been added to the docs:

https://twitter.github.io/heron/docs/operators/deployment/schedulers/kubernetes/
https://github.com/twitter/heron/blob/master/website/content/docs/operators/deployment/schedulers/kubernetes.md

I'd recommend closing this PR.

**Comment by lucperkins** [comment]
The Kubernetes support in Heron has changed dramatically from the initial iteration and we now have documentation for that new setup here: https://twitter.github.io/heron/docs/operators/deployment/schedulers/kubernetes/.

**Pull request closed without merging**